src/VisualStudio/Core/Test.Next/project.json

```diff
@@ -2,18 +2,8 @@
   "dependencies": {
     "Newtonsoft.Json": "8.0.3",
     "Moq": "4.2.1402.2112",
```

Comment:
Hrmm.  I'm not sure i understand hte concern.  No types from these assemblies are used in this project.  So are you just using the entries in the json file just for another purpose?  If so... that feels strange and weird.